public/js/src/module/records-queue.js

```diff
@@ -11,16 +11,20 @@ import settings from './settings';
 import exporter from './exporter';
 import { t } from './translator';
 import $ from 'jquery';
+import formCache from './form-cache';
 
 let $exportButton;
 let $uploadButton;
 let $recordList;
 let $queueNumber;
 let uploadProgress;
 let finalRecordPresent;
-const autoSaveKey = `__autoSave_${settings.enketoId}`;
```

Comment:
This was moved to a function body primarily to aid testing, but also leaves us in a better position to reduce reliance on global state in the future.